0009-salt-working-groups.md

```diff
@@ -0,0 +1,121 @@
+
+#### Captain Guidelines
+
+- Captain will have merge rights to Salt once a PR has been reviewed and approved
+- A captain of one working group cannot be captain of any other salt working groups. But captain can be a participant of other working groups 
+- Captain can vote to choose the captain and/or participants of next term working group
+- Captains have a 3 successive term limit. Captains can take a gap after their 3 successive terms and come back to be a captain after a gap term
+- Starting from 2nd term working groups, the captains chosen need to have been a participant in at least one working group 
+- Once a captain’s term end and they are no longer the captain of the next term, their merge rights will be rescinded and transferred to new/incoming captain of the working group 
+- Captain’s need to follow the [Merging Guidelines](https://github.com/saltstack/salt-enhancement-proposals/); This is to ensure Salt project grows responsibly. 
+- In rare occasions and in consultation with Salt Core team, Captain can nominate a Vice Captain for the working group, to help assist during long leave of absence or to share responsibilities. Captain can also give merge access to said Vice Captain of their working group while ensuring they follow the Merging Guidelines.
+
+ 
+### Participant
+
+Participants form the core of each of the working groups. 
+
+#### Role of Participant 
+- Participants will attend monthly working group meetings for their respective working group 
+- Participants will get to work on exciting salt projects be it enhancements or issues that impacts thousands of salt users
+
+#### Participant Guidelines 
+- No restriction on the number of working groups a participant can be part of 
+- Once their term ends, a participant can move to different working group or continue in the same working group 
+- No term limits on how many times a Salt community member can be a participant of a working group 
+- Participants don’t have merge rights. But participants can get merge rights based on their captain’s recommendation 
+
+
+### Facilitator 
+
+Till the working groups mature, Saltstack will facilitate the working group meetings and there will be at least 1 SaltStack facilitator in every working group meeting
+
+#### Role of Facilitator 
+- Facilitator will work with Salt core team to identify the list of participants, communicate/invite them to working groups 
+- Facilitator will make sure everyone gets a chance to voice their opinions
+- Facilitator will make sure members stick to the goals of the meeting and doesn't get derailed into tangential conversations 
+- Facilitator sets up the monthly working group meetings and invites the right folks 
+- Facilitator makes the agenda clear before every meeting
+- Facilitator takes notes and share the action items after every meeting 
+- Facilitator makes sure working group members needs/interests are met 
+
```

Comment:
- `The facilitator` will work with `the` Salt core team to identify participants and communicate/invite them to `the` working groups 
- `The facilitator` will `ensure` everyone gets a chance to voice their opinions
- `The facilitator` will `ensure` members `adhere` to the goals of the meeting and `that the meeting doesn't devolve` into tangential conversations 
- `The facilitator` sets up the monthly working group meetings and invites the appropriate participants
- `The facilitator` makes the agenda clear before every meeting
- `The facilitator` takes notes and `shares` the action items after every meeting 
- `The facilitator ensures the` working group members needs/interests are met